### PR TITLE
Backport 1.6.1: "vault operator usage" CLI for client count reporting (#10365)

### PR DESCRIPTION
**Pull request by mgritter**
* Working draft of CLI command.
* Sort order, robustness checking.
* Text edits and check of queries_available.
* Added changelog.

### CODE REVIEW

File: command/operator_usage.go [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+func (c *OperatorUsageCommand) AutocompleteFlags() complete.Flags {
+	return c.Flags().Completions()
+}
+
+func (c *OperatorUsageCommand) Run(args []string) int {
+	f := c.Flags()
+
+	if err := f.Parse(args); err != nil {
+		c.UI.Error(err.Error())
```
Comment (swayne275) [inline command/operator_usage.go:84]: i'm assuming that `c.UI.<loglevel>` calls output strings to the user? If so, do we want to expose the internal workings of the system by printing the error given back from function calls?

File: command/operator_usage.go [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+
+	client, err := c.Client()
+	if err != nil {
+		c.UI.Error(err.Error())
+		return 2
+	}
+
+	resp, err := client.Logical().ReadWithData("sys/internal/counters/activity", data)
+	if err != nil {
+		c.UI.Error(fmt.Sprintf("Error retrieving client counts: %v", err))
```
Comment (swayne275) [inline command/operator_usage.go:104]: same as above

**Review by swayne275 — APPROVED** [review]
Looks good overall.

I guess I'm not sure of the c.UI.Error/Warn calls. From the code it appears both go to stdout for the CLI. In this code it looks like (at least in general) the Warn messages are formatted with a particular style (caps and punctuation), but the Error messages are not. Do we want those to be in sync with each other and of a specific format for UX? Also, do we want to expose errors from called functions via stderr (rather than some user facing message on std err and the more specific details in a log)?

File: command/operator_usage.go (hunk 2 above)
Comment (mgritter) [reply command/operator_usage.go:104]: This is how the client handles most errors from Vault, I think. If there's a better example I'll copy it (but that should be addressed in the main branch, not this backport PR.)

Comment (swayne275) [reply command/operator_usage.go:104]: oh okay, wasn't sure what the ux was

**Pull request merged**